retroshare-gui/src/gui/settings/RSPermissionMatrixWidget.h

```diff
@@ -34,7 +34,7 @@
 #define SCALE_WIDTH   75  /** Width of the scale */
 
 #define BACK_COLOR       Qt::white
-#define FOREGROUND_COLOR Qt::black
+#define FOREGROUND_COLOR Qt::gray
```

Comment:
I do not like it. With built-in in light mode and standard light, it is not easy readable and gives me eye cancer. Built-in in dark mode looks good, the same in standard dark.